euphonic/cli/utils.py

```diff
@@ -439,12 +438,8 @@ def _brille_calc_modes_kwargs(args: Namespace) -> Dict[str, Any]:
     if args.n_threads is None:
         # Nothing specified, allow defaults
         return dict()
-    else:
-        if args.n_threads > 1:
-            parallel = True
-        else:
-            parallel = False
-        return dict(useparallel=parallel, threads=args.n_threads)
+
+    return dict(useparallel=(args.n_threads > 1), threads=args.n_threads)
```

Comment:
This one is hand-tuned rather than the ruff suggestion (which still had a needless if/else).